app/src/main/res/layout/preferences_page_layout.xml

```diff
@@ -17,22 +17,65 @@
       android:textSize="55sp"
       android:textStyle="bold" />
 
-  <ListView
-      android:id="@+id/preferencesListView"
-      android:layout_width="fill_parent"
-      android:layout_height="wrap_content"
-      android:cacheColorHint="#00000000"
-      android:drawSelectorOnTop="false"
-      android:layout_weight="5">
-
-  </ListView>
-
-  <Button
-      android:id="@+id/deleteAccount"
-      android:layout_width="200dp"
-      android:layout_height="50dp"
-      android:text="@string/delete_my_account"
+
+  //Button for deleting an account.
+
+  <LinearLayout
+      android:layout_width="379dp"
+      android:layout_height="200dp"
+      android:layout_centerInParent="true"
       android:layout_gravity="center"
-      android:layout_weight="1" />
+      android:layout_marginTop="16dp"
+      android:background="@drawable/coner">
```

Comment:
Please get rid of this background

Reply:
To be honest someone can just remove this later, I am just going to merge right now, this seems like a small fix.